main.go

```diff
@@ -119,6 +126,16 @@ func main() {
 	stringFlagOrEnv(&neuronMonitorImage, "neuron-monitor-image", "RELATED_IMAGE_NEURON_MONITOR", fmt.Sprintf("%s:%s", neuronMonitorImageRepository, v.NeuronMonitor), "The default Neuron monitor image. This image is used when no image is specified in the CustomResource.")
 	pflag.Parse()
 
+	// set instrumentation cpu and memory limits in environment variables to be used for default instrumentation
+	autoInstrumentationConfig := map[string]map[string]string{"java": {"cpu": "500m", "memory": "64Mi"}, "python": {"cpu": "500m", "memory": "32Mi"}, "dotnet": {"cpu": "500m", "memory": "128Mi"}}
```

Comment:
I see limits, but are we planning on making requests configurable as well?

Reply:
Yes, added.